### ISSUE TITLE
Life Domain Error in Disciple of Life 

### ISSUE THREAD

**Issue by dafrca** [Sklore/HL_DD_5e_Colab#662]
EDIT: Sorry Team, for some reason I thought this was not in the SRD. Please forgive me for wasting your time. 

In the Hero Lab pack, the Disciple of Life from the Life Domain says "Healing spells of 2nd level or higher heal additional...."

But in the Players Handbook it says; "Whenever you use a spell of 1st level or higher to restore hit points ...."

They both agree on what happens, just not what level spells are impacted. 

In other words, it should say 1st level spells not 2nd level spells.

**Comment by DarkHayZe** [comment]
This seems to be an error on Hero Labs end. Unfortunately not something we can take care of.

**Comment by dafrca** [comment]
So sorry, I thought it was part of the pack. Sorry to have bugged you guys. 

Daniel

**Comment by DarkHayZe** [comment]
No worries 

_Sent from my Huawei Nexus 6P using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by DarkHayZe** [comment]
So I did email Wolf-Lair about this, and I noticed while doing so, if you click on the question mark next to it, it does give the correct stats. So it is just the abilit box that is wrong.

![image](https://cloud.githubusercontent.com/assets/17392134/26658987/1c2d3196-4633-11e7-9bd2-c21d930081c8.png)

**Comment by dafrca** [comment]
Cool, thanks for the update as well. :-)
